packages/actor-rdf-parse-html/lib/ActorRdfParseHtml.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  ActorRdfParse,
+  ActorRdfParseFixedMediaTypes,
+  IActionRdfParse,
+  IActionRootRdfParse,
+  IActorOutputRootRdfParse,
+  IActorRdfParseFixedMediaTypesArgs,
+  IActorRdfParseOutput,
+  IActorTestRootRdfParse,
+} from "@comunica/bus-rdf-parse";
+import {ActionContext, Actor, IActorTest, Mediator} from "@comunica/core";
+import {Readable} from "stream";
+
+/**
+ * A HTML RDF Parse actor that listens on the 'rdf-parse' bus.
+ *
+ * It is able to extract JSON-LD from HTML files and parse the JSON-LD based RDF serializations
+ * and announce the presence of them by media type.
+ */
+// Gebaseerd op JSON LD parser
+export class ActorRdfParseHtml extends ActorRdfParseFixedMediaTypes {
+
+  public mediatorRdfParse: Mediator<Actor<IActionRootRdfParse, IActorTestRootRdfParse,
+    IActorOutputRootRdfParse>, IActionRootRdfParse, IActorTestRootRdfParse, IActorOutputRootRdfParse>;
+
+  constructor(args: IActorRdfParseFixedMediaTypesArgs) {
+    super(args);
+  }
+
+  public async runHandle(action: IActionRdfParse, mediaType: string, context: ActionContext):
+      Promise<IActorRdfParseOutput> {
+
+    const htmlString: string = await require('stream-to-string')(action.input);
+
+    // JSON-LD extraction
+    let jsonString: string = '';
```

Comment:
Not sure if this code will work properly when there are multiple JSON-LD script tags available. Be sure to add a test for that.